style.css

```diff
@@ -796,4 +781,133 @@ input[type='submit']:active {
   * {
     margin: auto;
   }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  .desktop-menu li a {
+    color: #344563;
+    text-align: center;
+    font-family: Poppins, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-decoration: none !important;
```

Comment:
- Hey, let's try to add the rule without using the !important rule as a guide in the microverse list of the [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). Plese, kindly remove it.